src/main/java/net/fabricmc/loom/build/nesting/IncludedJarFactory.java

```diff
@@ -58,6 +62,9 @@
 
 public final class IncludedJarFactory {
 	private final Project project;
+	private static final Logger LOGGER = LoggerFactory.getLogger(IncludedJarFactory.class);
+	private static final String semverRegex = "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$";
+	private static final Pattern pattern = Pattern.compile(semverRegex);
```

Comment:
Nit: these should be in `CONSTANT_CASE`

Reply:
Resolved in https://github.com/FabricMC/fabric-loom/pull/1075/commits/67e0fec0214ba2e0d5f34f7b55833383b53d24ac, thanks for the note :)